openqa-mail-notification/mail_notification.rb

```diff
@@ -85,7 +85,7 @@ def send_notification(to, subject, message)
   end
 
   last_build = DateTime.parse(build['t_finished'])
-  frequency_minutes_ago = DateTime.now - (FREQUENCY / 1440.0)
+  frequency_minutes_ago = DateTime.now.new_offset(0) - (FREQUENCY / 1440.0)
```

Comment:
[`0` is the default value](https://ruby-doc.org/stdlib-2.5.1/libdoc/date/rdoc/DateTime.html#method-i-new_offset), so this code could be changed to:

```suggestion
  frequency_minutes_ago = DateTime.now.new_offset - (FREQUENCY / 1440.0)
```

Reply:
hm I find this less clear TBH